definitions/variable/energy/secondary-energy-non-electricity.yaml

```diff
@@ -0,0 +1,115 @@
+- Secondary Energy|Liquids:
+    definition: Total production of refined liquid fuels from all energy sources,
+      including oil products, synthetic fossil fuels from gas and coal, and biofuels
+    unit: EJ/yr
+- Secondary Energy|Liquids|Biomass:
+    description: Production of liquid fuels from biomass
+    unit: EJ/yr
+- Secondary Energy|Liquids|{Primary Fossil Fuel}:
+    description: Production of fossil liquid synfuels from {Primary Fossil Fuel}
+    unit: EJ/yr
+- Secondary Energy|Liquids|Electricity:
+    description: Production of liquid fuels from electricity (e-fuels)
+    unit: EJ/yr
+- Secondary Energy|Liquids|Other:
+    description: Production of liquids fuels from sources that do not fit any other category
+    unit: EJ/yr
+
+- Secondary Energy|Solids:
+    definition: Total production of solid energy carriers, including coal, briquettes,
+      coke, and wood pellets
+    unit: EJ/yr
+- Secondary Energy|Solids|Biomass:
+    description: Production of solid secondary energy carriers from biomass
+      (e.g., commercial charcoal, wood chips, wood pellets).
+      Use of tradional bioenergy is excluded.
+    unit: EJ/yr
+- Secondary Energy|Solids|Fossil:
+    description: Production of solid secondary energy carriers from fossils
+      (e.g., briquettes, coke)
+    unit: EJ/yr
+
+- Secondary Energy|Gases:
+    definition: Total production of gaseous fuels, including natural gas and biogas,
+      not including hydrogen
+    unit: EJ/yr
+- Secondary Energy|Gases|Biomass:
```

Comment:
Do we not consider CCS here because gas production does not allow for it?

Reply:
My intuition is that the gas (methane) still contains the carbon molecules, hence CCS would be accounted for at the point of combustion (e.g., in the power plant when generating electricity).